### DISCUSSION TITLE
related template: CONTRIBUTING.md

### DISCUSSION THREAD

**Discussion by jokeyrhyme** [olivierlacan/keep-a-changelog#494] [Ideas]
Hiya folks!

I put together a template CONTRIBUTING.md file for my own use, and felt it could be useful for others:
- https://github.com/jokeyrhyme/keep-contributing

If you think it's appropriate, it might be worth a "related resources" link or something.

Comments are very welcome!

**Comment by cdeszaq** [comment]
I think that resource is fantastic, and cross-linking is a great idea between these two projects.

**Comment by olivierlacan** [comment]
This is a really cool project. We don't really have an "Resources for maintainers" section yet. Can you propose one @cdeszaq or @jokeyrhyme?

**Comment by jokeyrhyme** [comment]
I highly recommend this, too: https://github.com/blog/2380-new-community-tools
